File: docs/components_page/components/layout/order_offset.jl
using DashBootstrapComponents, DashHtmlComponents

row = html_div([
    dbc_row(
        dbc_col(
            html_div("A single, half-width column"),
            width = Dict("size" => 6, "offset" => 3),
        ),
    ),
    dbc_row([
        dbc_col(
            html_div("The last of three columns"),
            width = Dict("size" => 3, "order" => "last", "offset" => 1),
        ),
        dbc_col(
            html_div("The first of three columns"),
            width = Dict("size" => 3, "order" => 1, "offset" => 2),
        ),
        dbc_col(
            html_div("The second of three columns"),
            width = Dict("size" => 3, "order" => 5),
        ),
    ]),
]);

File: docs/components_page/components/layout/order_offset.R
library(dashBootstrapComponents)

row <- div(
  list(
    dbcRow(
      dbcCol(
        div("A single, half-width column"),
        width = list(size = 6, offset = 3)
      )
    ),
    dbcRow(
      list(
        dbcCol(
          div("The last of three columns"),
          width = list(size = 3, order = "last", offset = 1)
        ),
        dbcCol(
          div("The first of three columns"),
          width = list(size = 3, order = 1, offset = 2)
        ),
        dbcCol(
          div("The second of three columns"),
          width = list(size = 3, order = 5)
        )
      )
    )
  )
)
